azure-pipelines.yml

```diff
@@ -286,6 +286,25 @@ jobs:
         channels:
           - main
 
+  # Affinitized Scenario benchmarks (Initially just PDN)
+  - template: /eng/performance/build_machine_matrix.yml
+    parameters:
+      jobTemplate: /eng/performance/scenarios.yml
+      buildMachines:
+        - win-x64
+        - win-arm64
+      isPublic: false
+      jobParameters:
+        kind: scenarios
+        projectFile: scenarios_affinitized.proj
+        channels:
+          - main
+        affinity: '85'
```

Comment:
This should be `0x55` for x64 and `0xF` for arm64.

Reply:
The value is currently passed in as a decimal so the 85 corresponds to 0x55. Other than not specifying the alternate cores, is there a specific reason to use 0xF over 0x55? Using the same for both will make some of the downstream report generation more clean, if possible.

Reply:
I honestly not sure if specifying one way or the other makes any difference. lets go with `0x55` for now. Just to confirm these settings are also applicable for ml.net and roslyn, right?

Reply:
Yup, both ml.net and roslyn are already using 0x55.